src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/nocodeinfile/Example1.java

```diff
@@ -9,7 +9,6 @@
 // violation 8 lines above
 
 // xdoc section -- start
-// the violation is on first line of file
```

Comment:
`/* the violation is on first line of file */`

same for all files in nocodeinfile folder

Reply:
fyi - https://checkstyle.org/checks/misc/nocodeinfile.html#Example2-code

Reply:
Done